ts/torch_handler/vision_handler.py

```diff
@@ -20,6 +20,9 @@ def initialize(self, context):
         super().initialize(context)
         self.ig = IntegratedGradients(self.model)
         self.initialized = True
+        properties = context.system_properties
+        if not properties.get("limit_max_image_pixels") :
+            Image.MAX_IMAGE_PIXELS = None
```

Comment:
Here, we're setting value to None, however, it does not seem to be clear what happens if this value is not None, does PILLOW automatically pick it up from context?

Reply:
Same as the above, the detailed information is provided in pillow link